doc/examples/nrop.ocpci.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: nodetopology.openshift.io/v1alpha1
```

Comment:
I understand that `v1alpha1` apiversion would be converted to `v1` apiversion so this is not wrong but shouldn't we update this to the latest supported version i.e. `v1` ? I guess the same applies to the `nrop.yaml `manifest below.